exporter.py

```diff
@@ -8,14 +8,19 @@
 import sys
 
 import paho.mqtt.client as mqtt
-from prometheus_client import Gauge, start_http_server
+from prometheus_client import Gauge, Counter, start_http_server
 
 logging.basicConfig(level=os.environ.get("LOG_LEVEL", "INFO"))
 LOG = logging.getLogger("mqtt-exporter")
 PREFIX = os.environ.get("PROMETHEUS_PREFIX", "mqtt_")
 
 # global variable
 prom_metrics = {}  # pylint: disable=C0103
+topic_label = os.environ.get("TOPIC_LABEL", "topic")
```

Comment:
when set at global level, a constant should be in uppercase
and please move this line higher (before "global variable" as this is not a variable.

I am surprised pylint did not catch it.